### PR TITLE
Remove ref to fs.config.

### PR DESCRIPTION
**Pull request by jburel**
Remove reference to the fs.config file
We no longer use it.

--no-rebase

### CODE REVIEW

File: components/insight/SRC/org/openmicroscopy/shoola/env/data/DataServicesFactory.java [hunk 1]
```diff
@@ -45,11 +41,6 @@
 
 import javax.swing.JDialog;
 import javax.swing.JFrame;
-
-//Third-party libraries
-
-
-//Application-internal dependencies
```
Comment (mtbc) [inline components/insight/SRC/org/openmicroscopy/shoola/env/data/DataServicesFactory.java]: don't know if you wanted to remove this but not the Java imports comment

**Comment by joshmoore** [comment]
Both the Mac_Java7+ and the Linux zips opened fine. Reviewing changes now...

**Comment by joshmoore** [comment]
Do any of these files deserve removing?
```
$ git ls-files | grep insight | grep -i fs | wc -l
     100
```

**Comment by jburel** [comment]
Ironically when some of the work started, the agent was named "fsimporter"
I have kept the filesystem view and related files if we decide to offer view of the repo itself (discussed a while ago!)

**Comment by jburel** [comment]
not planning further changes on this PR.

**Pull request merged**